protein_ligand/README.md

```diff
@@ -135,6 +136,27 @@ We also accept non-ranked submissions, which we will not formally judge. These a
 
 If multiple submissions are incorrectly provided as "ranked" by a single participant, we will judge only one of them; likely this will be the first submitted, but it may be a random submission.
 
+**Submission format**: Your submissions must consist of a completed submission description following [our format](stage2_submission_template.txt) (with a file name beginning with the characters `PHIP2_2` and ending with `.txt`) and a .tar.gz `pose.tgz` file providing your pose predictions compressed and archived in a single directory. This file is used to submit one set of predicted protein-ligand poses, where up to five poses are permitted for each protein-ligand pair (if multiple poses are submitted, subsequent poses are given decreasing weight in judging). Each pose tgz file must contain, for each ligand, a minimum of one and up to 5 protein structure PDB files and 5 corresponding ligand poses (in MDL `.mol`, format, `.sdf` format, or `.mol2` format) predicted by a method described in the separately uploaded pose prediction description file.  
+
+**Each pose prediction must be provided in the form of a protein structure PDB file and a corresponding ligand structure file (`.sdf`, `.mol2`, or MDL `.mol`) with all-atom 3D atomic coordinates for the pose, where the coordinates in the protein PDB file and the ligand molfile are in the same frame of reference**. Any ligand coordinates provided in PDB format or included in the protein PDB files will be ignored. You may treat the protein as rigid or flexible, but you must rotationally and translationally superimpose all of your final structure predictions, onto the [reference protein structure provided](PHIPA_C2_Apo.pdb) in the challenge data package in order to facilitate evaluation of your predictions.
+
+The file names of your pose prediction protein PDB and ligand mol files must be constructed as follows:
+
+```
+PHIP2-<LigandID>-<poseRank>.pdb
+PHIP2-<LigandID>-<poseRank>.mol (or .sdf or .mol2)
```

Comment:
I am surprised we are accepting three different file types. Is it necessary? Wouldn't it be simpler for the analysis if we only allowed mol files?

Reply:
I can handle it with the same OpenEye code without headache. D3R did just .mol files, but I have a personal preference for sdf or mol2, so I was thinking it'd be OK to support all three.